scripts/analysis.py

```diff
@@ -663,7 +806,51 @@ def plot_power(cursor):
     con = lite.connect(file)
     with con:
         cur = con.cursor()
-        print(snf(cur))
-        plot_power(cur)
-        plot_in_out_flux(cur, 'source', False, 'source vs time', 'source')
-        plot_in_out_flux(cur, 'sink', True, 'isotope vs time', 'sink')
+        # print(snf(cur))
+        # plot_power(cur)
+        # plot_in_out_flux(cur, 'source', False, 'source vs time', 'source')
+        # plot_in_out_flux(cur, 'sink', True, 'isotope vs time', 'sink')
+        init_year, init_month, duration, timestep = get_sim_time_duration(cur)
+        
+        #    waste_dict ['Reactor'] = uox_waste
+        #    waste_dict ['Enrichment'] = tailing
+        #    waste_dict ['Separations'] = reprocess waste (FP, MA)
+        #    pile_dict ['Mixer'] = tailing
+        #    pile_dict2 ['Separation'] = reprocessed U
+        
+        waste_dict = total_waste_timeseries(cur)
+        multi_line_plot(waste_dict, timestep,
+                        'Years', 'Mass[MTHM]',
+                        'Total Waste Mass vs Time',
+                        'total_Waste',
+                        init_year)
+
+        fuel_dict = fuel_usage_timeseries(cur, ['uox', 'mox'])
+        stacked_bar_chart(fuel_dict, timestep,
+                          'Years', 'Mass[MTHM]',
+                          'Total Fuel Mass vs Time',
+                          'total_fuel',
+                          init_year)
+        final_stockpile(cur, 'Mixer')
+        final_stockpile(cur, 'Separations')
+
+        try:
+            pile_dict = get_stockpile(cur, 'Mixer')
+            multi_line_plot(pile_dict, timestep,
+                            'Years', 'Mass[MTHM]',
+                            'Tailings left over in Mixer vs Time',
+                            'Total_Stockpile', init_year)
+            pile_dict2 = get_stockpile(cur, 'Separations')
```

Comment:
if you have two variables with the same name, compelling you to name them var1 and var2, it's a sign that the name is not meaningful. can I recommend "separations_stockpile" or similar here?